dist/tools/tapsetup/tapsetup

```diff
@@ -17,6 +17,8 @@ usage() {
     echo "Options:" >&2
     echo "   -c [<num>], --create [<num>]: Create <num> tap interfaces (default: 2)" >&2
     echo "   -d, --delete:                 Delete all interface" >&2
+    echo "   -l <iface>, --list <iface>:   List all interfaces of the bridge <iface> belongs" >&2
+    echo "                                 to (including the bridge itself)." >&2
```

Comment:
```suggestion
    echo "   -l <iface>, --list <iface>:   List all interfaces connected the bridge <iface> " >&2
    echo "                                 (including the bridge itself)." >&2
```
?

Reply:
Mh no. You can also use `tapsetup -l tap0` and it will list

```
tapbr0:
- tap0
- tap1
```

like it will for `tapsetup -l tapbr0` this is e.g. useful for a portable version of functions like those:

https://github.com/RIOT-OS/RIOT/blob/b2da5d0687ea72dc235f35994f20e4e7071b7040/tests/gnrc_ipv6_ext/tests/01-run.py#L570-L576

Reply:
But the sentence is still missing a word

Reply:
> If `<iface>` belongs to a bridge, list the bridge and all interfaces that belong to it.

Yea that sounds more clear.

Reply:
Done